src/strings/csv/en.csv

```diff
@@ -126,6 +127,12 @@ APPLICATION_PRESCREEN_SUCCESS_SUBTITLE,"Your Pre-screen is complete, and you qua
 APPLICATION_PRESCREEN_SUCCESS_TITLE,Success! Your Pre-screen qualifies.,
 APPLICATION_RESTORABLE_LAND,Application Restorable Land (ha),
 APPLICATION_STATUS,Application Status,
+APPLICATION_STEP_APPLICATION_DESCRIPTION,"For the full Accelerator Program Application, you will answer general questions about your project as well as forest restoration, community impact, financial and legal questions.",
+APPLICATION_STEP_APPLICATION_NAME,Application,
+APPLICATION_STEP_DUEDILIGENCE_DESCRIPTION,"Once your Application is submitted, the Accelerator team reviews your answers, potentially asks for updates and proceeds to voting and scoring your Application.",
+APPLICATION_STEP_DUEDILIGENCE_NAME,Due Diligence,
```

Comment:
Nit, but it would be nice for `DUEDILIGENCE` to be `DUE_DILIGENCE` since it's two distinct words